l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -98,21 +98,57 @@ def message_route(self, message, message_dict, model=None, thread_id=None,
                 return []
 
         elif self._context.get('fetchmail_server_id', False):
-            fetchmail_server_id = self.env['fetchmail.server'].browse(
+            # This is not an email coming from SDI
+            fetchmail_server = self.env['fetchmail.server'].browse(
                 self._context['fetchmail_server_id'])
-            if fetchmail_server_id.is_fatturapa_pec:
+            if fetchmail_server.is_fatturapa_pec:
                 att = self.find_attachment_by_subject(message_dict['subject'])
                 if att:
+                    # This a PEC response (CONSEGNA o ACCETTAZIONE)
+                    # related to a message sent to SDI by us
                     message_dict['model'] = 'fatturapa.attachment.out'
                     message_dict['res_id'] = att.id
                     self.clean_message_dict(message_dict)
                     self.env['mail.message'].with_context(
                         message_create_from_mail_mail=True).create(
                             message_dict)
                 else:
-                    # todo send email for non-routable pec mail
-                    _logger.error('Can\'t route PEC E-Mail with Message-Id: {}'
-                                  .format(message.get('Message-Id')))
+                    _logger.info(
+                        'Can\'t route PEC E-Mail with Message-Id: {}'.format(
+                            message.get('Message-Id'))
+                    )
+                    if fetchmail_server.e_inv_notify_partner_ids:
+                        self.env['mail.mail'].create({
+                            'subject': _(
+                                "PEC message [%s] not processed"
+                            ) % message.get('Subject'),
+                            'body_html': _(
+                                "<p>"
+                                "PEC message with Message-Id %s has been read "
+                                "but not processed, as not related to an "
+                                "e-invoice.</p>"
+                                "<p>Please check PEC mailbox %s, at server %s,"
+                                " with user %s</p>"
+                            ) % (
+                                message.get('Message-Id'),
+                                fetchmail_server.name, fetchmail_server.server,
+                                fetchmail_server.user
+                            ),
+                            'recipient_ids': [(
+                                6, 0,
+                                fetchmail_server.e_inv_notify_partner_ids.ids
+                            )]
+                        })
```

Comment:
Are you sending the mail?
```suggestion
                        }).send()
```

Reply:
Yes, email is sent by Email Queue Manager cron